yt_dlp/extractor/tiktok.py

```diff
@@ -173,8 +183,9 @@ def _call_api(self, ep, query, video_id, fatal=True,
 
         max_tries = len(self._APP_INFO_POOL) + 1  # _APP_INFO_POOL + _APP_INFO
         for count in itertools.count(1):
-            self.write_debug(str(self._APP_INFO))
             real_query = self._build_api_query(query)
+            device_id = 'PRIVATE' if self._KNOWN_DEVICE_ID else real_query['device_id']
```

Comment:
It'll still show in the verbose head. So what's the point?

Reply:
true